composer.json

```diff
@@ -22,9 +22,14 @@
     "require": {
         "php": ">=5.3.2",
         "symfony/doctrine-bridge": "~2.1",
+        "symfony/console": "~2.1",
```

Comment:
This bundle should not depend on Console, add it as require-dev if you must, like in the library: https://github.com/doctrine/data-fixtures/blob/master/composer.json.